### PR TITLE
Use small angular offsets when determining compass properties

### PR DESCRIPTION
**Pull request by astrofrog**
### Description

While debugging an issue with GWCS coordinates with @bmorris3, I found that one issue is that if an image with a bounding box is used, and if we were to not reset the bounding box to ``None`` as we do currently in jadviz, the compass properties will fail in some cases - this is because before this PR, the pixel scale along the N and E directions was determined by adding 1 deg to the RA and Dec, but this is too large as this may then fall outside the image and return NaN (with https://github.com/spacetelescope/gwcs/pull/497).

In any case, I think it makes sense to use small angular offsets when trying to do this calculation regardless of whether we decide to not disable the bounding box etc. I don't think there is any downside to using small offsets, and it will be more future-proof.

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Review by pllim — APPROVED** [review]
Seems reasonable and does fix the N-E arrows on ImvizDitheredExample notebook. Thanks!

**Comment by pllim** [comment]
There are 2 failures that just need their results updated.

File: jdaviz/configs/imviz/wcs_utils.py [hunk 1]
```diff
@@ -98,18 +98,26 @@ def calc_compass(image_wcs, x, y, len_deg_e, len_deg_n):
 
 
 def calc_compass_radius(image_wcs, x, y, radius_px):
```
Comment (pllim) [inline jdaviz/configs/imviz/wcs_utils.py:100]: @ejeschke , this function was actually from `ginga`:

https://github.com/ejeschke/ginga/blob/7e61d9772ec1e57c332f1e2c0280d308972640c4/ginga/util/wcs.py#L762-L764

Are you interested to have a similar patch over at `ginga`?

Comment (ejeschke) [reply jdaviz/configs/imviz/wcs_utils.py:100]: That looks like a good change, so sure!

Comment (pllim) [reply jdaviz/configs/imviz/wcs_utils.py:100]: OK please see https://github.com/ejeschke/ginga/pull/1102 . Thanks!

**Pull request merged**